docs/source/documentation/platform/infrastructure/run-data-product-in-platform.html.md.erb

```diff
@@ -1,48 +1,65 @@
 ---
 owner_slack: "#data-platform-notifications"
-title: Run data product in Data Platform (POC)
-last_reviewed_on: 2023-04-04
+title: Data as a Product in Data Platform (PoC)
+last_reviewed_on: 2023-09-27
 review_in: 6 months
 weight: 15
 ---
 
 # <%= current_page.data.title %>
 
-## Context
-We want a compute process that is scalable and event driven to run the data product. The data product will be sent to Data Platform for processing. After processing, the data should be available in a suitable format accessible to data consumers. The processing will include receiving raw data, applying transformations, cleansing and finally storing data in a correct location and format.
+## Data as a Product: Vision
 
-<strong>The architecture will be built in the `data-platform-development`, a sandbox account within MoJ's Modernisation Platform, which is deployed using the [modernisation-platform-environments](https://github.com/ministryofjustice/modernisation-platform-environments) repository</strong>
+We want to design the Data Product management process around a user-facing API.
+This API will allow users to register, update, and retrieve information about a Data Product.
 
-## Data Producer Configuration Flow
+The data sent to this API as part of registering or updating a data product will flow through standard Data Lake zones: Landing -> Raw -> Curated.
 
-![Data platform general diagram](../images/data-platform-configuration-flow.png "Data platform configuration Flow")
+Each Data Product will require a [specification](https://github.com/ministryofjustice/data-platform-products/blob/main/data-products/_docs/product-specification.md) (still under refinement) alongside [other definitions](https://github.com/ministryofjustice/data-platform-products/tree/main/data-products#defining-a-data-product).
 
-**For now** - Data product configuration will be sent to the Data Platform S3 bucket via [GitHub Actions](https://github.com/ministryofjustice/data-platform-products/blob/main/.github/workflows/data-product-example-2-poc.yaml).
+Data Products:
 
-The data product configuration (Zip file) will be stored in the following structure:
-> `${DATA_PLATFORM_PRODUCTS}/code/${DATA_PLATFORM_PRODUCT_NAME}/${ZIPFILE}.zip`
+* May be multi-component, e.g. consisting of multiple tables
+* Can consist of structured or unstructred data (or a combination)
+* Are controlled and updated by Data Owners, Data Stewards, and Data Custodians.
```

Comment:
These terms should definitely be defined on first use (I'm not sure of the exact meanings myself, and they aren't defined in our [team glossary](https://docs.google.com/spreadsheets/d/1O1EjO96lyqDbyuzFWU8SNBky3b7tKM8dccZVS7-_t9g/edit#gid=1105700365))

Reply:
@jemnery whereabouts can I find references to what we mean in Data Platform by these terms?

Reply:
You're right, I'm fairly sure we haven't defined the Data Steward and Data Custodian roles. In fact those definitions are really up to the DIP team, so let's not create a blocker waiting for those. There's a suggested edit in the diff.